src/gui/tray/unifiedsearchresultslistmodel.h

```diff
@@ -71,7 +71,7 @@ class UnifiedSearchResultsListModel : public QAbstractListModel
     [[nodiscard]] QString errorString() const;
     [[nodiscard]] bool waitingForSearchTermEditEnd() const;
 
-    Q_INVOKABLE void resultClicked(const QString &providerId, const QUrl &resourceUrl) const;
+    Q_INVOKABLE void resultClicked(const QString &providerId, const QUrl &resourceUrl, const QString &subline, const QString &title) const;
```

Comment:
```suggestion
    Q_INVOKABLE void resultClicked(const QString &providerId, const QUrl &resourceUrl, const QString &subLine, const QString &title) const;
```
we use camel case style

Reply:
no it is fine
thanks for the API doc, I would not have checked this